common/consumer-rules.pro

```diff
@@ -0,0 +1,2 @@
+-dontwarn com.google.errorprone.annotations.CanIgnoreReturnValue
+-dontwarn com.google.errorprone.annotations.Immutable
```

Comment:
Looks like the new R8 full mode is not compatible with the latest version of Tink-Android... Currently that library is unused until we actually finish Trusted entitlements. For now, I checked and these 2 annotations seem to only be used for internal checks of the library, so it should be safe to ignore them... In any case, I reported this in Tink-Java: https://github.com/tink-crypto/tink-java/issues/7 so they can hopefully add the rules in the library itself.